.gitignore

```diff
@@ -72,6 +72,7 @@ instance/
 # Sphinx documentation
 docs/_build/
 doc/_build/
+docs/source/_build/
```

Comment:
Can you revert these changes and make a separate PR if this is relevant? This keeps the git history clean.